docs/docs/training-data-format.mdx

```diff
@@ -381,11 +381,11 @@ stories:
 Each step can be one of the following:
 
   - A [user message](#user-messages), represented by **intent** and **entities**.
-  - An [or statement](#or-statement), which includes two or more user messages under it
-  - A bot [action](#actions)
-  - A [form](#forms)
-  - A [slot was set](#slots) event
-  - A [checkpoint](#checkpoints), which connects the story to another story
+  - An [or statement](#or-statement), which includes two or more user messages under it.
+  - A bot [action](#actions).
```

Comment:
```suggestion
  - A bot [action](#actions).
  - A bot [utterance](#end-to-end-training)
```

Reply:
this is left the same intentionally, see discussion above